autogpt/commands/file_operations.py

```diff
@@ -224,6 +224,68 @@ def write_to_file(filename: str, text: str, config: Config) -> str:
         return f"Error: {err}"
 
 
+@command(
+    "update_file",
+    "Update file",
+    '"filename": "<filename>",'
+    '"old_text": "<old_text>","new_text": "<new_text>",'
+    '"occurrence_index": "<occurrence_index>"',
+)
+def update_file(
```

Comment:
Hmm I'm not sure that the name of this command is reflective of what it does. I think `substitute_text` or something would be easier to understand.

Reply:
Discussed already, going to stick with update_file as it is consistent with other file commands.